backend/molgenis-emx2-webapi/src/test/java/org.molgenis.emx2.web/TestUtils.java

```diff
@@ -0,0 +1,86 @@
+package org.molgenis.emx2.web;
+
+import io.restassured.response.Response;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import net.lingala.zip4j.io.inputstream.ZipInputStream;
+import net.lingala.zip4j.model.LocalFileHeader;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+public class TestUtils {
```

Comment:
I am a bit surprised you didn't use the TableStore classes here.

Reply:
TableStore does not allow for easy parsing of sheet to String list ( only via Models objects). Having a separate test util makes it easy to write some tests helpers without the need to complicate the application code. Could add similar methods to application code, but if these methods are only there as test helpers i would argue against this, as is makes the api more complex for the caller